src/lib/__tests__/AnnotationControlsFSM-test.js

```diff
@@ -59,13 +59,21 @@ describe('lib/AnnotationControlsFSM', () => {
                 expect(annotationControlsFSM.getState()).to.equal(AnnotationState.NONE);
             });
         });
+
+        // Should reset state
+        it('should reset state if input is AnnotationInput.RESET', () => {
+            const annotationControlsFSM = new AnnotationControlsFSM(AnnotationMode.REGION);
```

Comment:
This test is under `NONE` group. The initial state should be `NONE`. The path `NONE -> NONE` still needs to be tested, right?

Reply:
Yes, that's correct